### ISSUE TITLE
In the model, the class and filename are the same, but I get an error.

### ISSUE THREAD

**Issue by tomomo** [louisl/CodeIgniter4-Standard#35]
Hello.

I set the file name and class name to be the same, but I get an error.

<img width="922" alt="スクリーンショット 2021-03-03 9 55 32" src="https://user-images.githubusercontent.com/4105561/109736119-46d65900-7c07-11eb-9812-75648983dbc5.png">

Is it a specification that capital letters other than the acronym and "Model" cannot be used?
（I was worried because the controller etc. can be used without problems.）

Is there a solution?

Thank you in advance.

**Comment by louisl** [comment]
The case for the file name and class name must match, your file has a lowercase 'P' in password. Rename your file from ```ResetpasswordModel.php``` to ```ResetPasswordModel.php```.

**Comment by louisl** [comment]
Bonus tip. If using git as it looks like you are, and you have committed that on a a case insensitive system like OS X, it's best to rename it something else like ```XResetPasswordModel.php``` then commit that, then rename to ```ResetPasswordModel.php``` and commit that or when you do a clone you'll get a file name of ```ResetpasswordModel.php``` because git stored that as the file name.

**Comment by tomomo** [comment]
Thank you for your reply.
I tried it as I was taught...

I renamed and committed, but it doesn't work.
I deleted it, committed it, made a new one and committed it, but that doesn't work either.

It was cured by creating a suitable remote repository, pushing it, and then cloning a new one.

Is it a `phpcs` issue, a `vscode or extension` issue, or a local `git` issue...
In any case, I've confirmed that this is not a problem with the "coding-standard" coding rules.

Thank you very much.
